regression/tests/studio/test_schedule_and_details_settings.py

```diff
@@ -94,3 +102,70 @@ def test_cancel_upload(self):
             self.settings_page.get_element(
                 '.wrapper-input input'
             ).get_attribute('value'))
+
+
+class ScheduleAndDetailsLinks(WebAppTest):
+    """
+    Tests for Studio's Setting page links.
+    """
+    def setUp(self):
+        super(ScheduleAndDetailsLinks, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.settings_page = SettingsPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.settings_page.visit()
+
+    def test_other_grading_link(self):
+        """
+        Verifies that user can click and navigate to Grading
+        """
+        link = 'href="/settings/grading/course-v1:ArbiRaees+AR-1000+fall"'
```

Comment:
This is another case where---on a different environment---tests would fail. I wonder if you can pull the course out as another variable... Or, if we want to think of another, scalable way to handle these situations. Perhaps we need to create courses as part of test setup? Or, perhaps we want a dedicated test data file (like a *.ini file) that we include when running tests?